buildSrc/src/main/kotlin/DependencyConfig.kt

```diff
@@ -48,6 +48,9 @@ fun Project.configureDependencies() {
         maven("https://jitpack.io") {
             name = "JitPack"
         }
+        maven("https://nexuslite.gcnt.net/repos/other/") {
```

Comment:
What is this repository for? i can only find info for some skyblock plugin type thing.

Reply:
Its for folialib for experimental folia support. Its going to be removed for a new standard paper api, but its only in 1.20(.2?)+ versions so it has to wait until be drop older versions